build/build-dotnet.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -ex
+
+VERSION=$1
+if echo ${VERSION} | grep 'trunk'; then
+    BRANCH=main
```

Comment:
How frequently are these builds produced?

Reply:
nightlies are not produced yet, we'll need to work on making the process of build to live is shorter, especially the nuget packaging which we somehow need to automate together with new versions. For now, let's see what users like/dislike/want/etc.